net/acme/files/acme-cbi.lua

```diff
@@ -57,6 +57,11 @@ u = cs:option(Flag, "update_uhttpd", translate("Use for uhttpd"),
                         "(only select this for one certificate)."))
 u.rmempty = false
 
+u = cs:option(Flag, "update_nginx", translate("Use for uhttpd"),
```

Comment:
Looks like a copy/paste error in the description

Reply:
yes mb ...